cmd/user-agent/src/main.js

```diff
@@ -82,7 +65,7 @@ async function ariesStartupOpts() {
 let defaultTrustBlocStartupOpts = {
     assetsPath: '/trustbloc-agent/assets',
     blocDomain: 'testnet.trustbloc.local',
-    walletMediatorURL: '',
+    walletMediatorURL: 'http://localhost:10093',
```

Comment:
It might be possible to load these as environment variables from the outside: https://cli.vuejs.org/guide/mode-and-env.html#modes

Reply:
I am loading it from the outside.I will rollback this change.